hypertrace-trace-enricher/hypertrace-trace-enricher/src/test/resources/configs/hypertrace-trace-enricher/application.conf

```diff
@@ -21,6 +21,11 @@ kafka.streams.config = {
 enricher {
   names = ["SpanTypeAttributeEnricher", "ApiStatusEnricher", "EndpointEnricher", "TransactionNameEnricher", "ApiBoundaryTypeAttributeEnricher", "ErrorsAndExceptionsEnricher", "BackendEntityEnricher", "HttpAttributeEnricher", "DefaultServiceEntityEnricher", "UserAgentSpanEnricher", "ExitCallsEnricher"]
 
+  executors = {
+    thread.pool.size = ${kafka.streams.config.num.stream.threads}
```

Comment:
wondering if we should expose this config?

Reply:
Added to helm values so that it can be configured from there.